jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/ServletPathSpec.java

```diff
@@ -26,17 +26,16 @@
 {
     private static final Logger LOG = Log.getLogger(ServletPathSpec.class);
 
-    private final String _declaration;
     private final PathSpecGroup _group;
     private final int _pathDepth;
     private final int _specLength;
     private final String _prefix;
     private final String _suffix;
 
     /**
-     * If a servlet or filter path mapping isn't a suffix mapping, ensure
-     * it starts with '/'
-     *
+     * Normalize servlet path spec:<ul>
```

Comment:
Ideally, this class would have been called UrlPatternPathSpec - servlets, filters and security constraints all refer to ``url-pattern``s.

Reply:
ideally yes